content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -61,9 +54,9 @@ datasource db {
 }
 ```
 
-You do not need to change your connection string. The `schema` value of your connection string is the default database schema that the Prisma Client connects to and uses for raw queries. All other Prisma Client queries will use the schema of the model that you are querying.
+You do not need to change your connection string. The `schema` value of your connection string is the default database schema that the Prisma Client connects to and uses for raw queries. All other Prisma Client queries will use the schema of the model or enum that you are querying.
 
-To designate a model as belonging to a specific database schema, add a `@@schema` attribute with the name of the database schema as a parameter. In the following example the `User` model is part of the `"base"` schema, and the `Order` model is part of the `"transactional"` schema:
+To designate a model or enum as belonging to a specific database schema, add a `@@schema` attribute with the name of the database schema as a parameter. In the following example the `User` model is part of the `"base"` schema, and the `Order` model and `Size` enum are part of the `"transactional"` schema:
```

Comment:
```suggestion
To designate a model or enum as belonging to a specific database schema, add a `@@schema` attribute with the name of the database schema as a parameter. In the following example the `Order` model and `Size` enum are part of the `"transactional"` schema and the `User` model is part of the `"base"` schema:
```

Suggestion to make it read a bit more smoothly

Reply:
Agree it reads a bit better, but I'd prefer to introduce the `base` schema first

Reply:
Ended up accepting another suggestion from Zach below, which still helps it read more smoothly